sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -322,9 +325,11 @@ public void testAssignments() {
 
   @Test
   public void testTokens() {
-    List<Token> tokens = kotlin("private fun foo() { }").metaData().tokens();
-    assertThat(tokens).extracting(Token::text).containsExactly("private", "fun", "foo", "(", ")", "{", "}");
-    assertThat(tokens).extracting(Token::isKeyword).containsExactly(true, true, false, false, false, false, false);
+    List<Token> tokens = kotlin("private fun foo() { 42 + \"a\" }").metaData().tokens();
+    assertThat(tokens).extracting(Token::text).containsExactly(
+      "private", "fun", "foo", "(", ")", "{", "42", "+", "\"", "a", "\"", "}");
```

Comment:
Not sure about this, but I would have expected to see a single token `"\"a\""` instead of the 3 tokens for the string literal `"a"`, as in the `SlangConverter`.
Although this is not directly related to this PR but tokens

Reply:
This is due to the fact that the Kotlin parser parses all string literals as templates.